interface/system/services.rst

```diff
@@ -164,22 +125,171 @@ NIS is a directory service to connect to a NIS server.
 
 To configure NIS, click on the **wrench** icon and submit the form with appropriate values as shown below.
 
-.. image:: /images/interface/system/services/nis-config.png
-   :width: 100 %
+.. image:: /images/interface/system/services/nis_config.png
+   :width: 70 %
    :align: center
 
 * **Domain**: NIS domain.
 * **Server**: IP address of NIS server.
 
+
+.. _ntp:
+
+NTP
+---
+
+NTP maintains system time in synchronization with Internet standard time server. This service must always be turned on.
+
+To configure NTP, you can specify the address of an Internet standard time server in the NTP configuration page.
+
+.. image:: /images/interface/system/services/ntp_config.png
+   :width: 70 %
+   :align: center
+
+
 NUT-UPS
 -------
 
-A (currently Beta) `Network UPS Tools <https://networkupstools.org/>`_ based service to orchestrate graceful system shutdown in the event of a power outage.
+A (currently Beta) `Network UPS Tools <https://networkupstools.org/>`_-based service to orchestrate graceful system shutdown in the event of a power outage.
 Please see our: :ref:`ups_setup` section for more details.
 
+
+.. _configure_replication:
+
+Replication
+-----------
+The Replication service leverages Btrfs send & receive features to replicate Shares to other Rockstor appliances (see :ref:`sharereplication` for more details).
+This can be used, for instance, to asynchronously and automatically replicate important shares on another Rockstor system.
+To use this feature, first configure the Replication service by clicking the **wrench** icon next to its name.
+
+.. image:: /images/interface/system/services/rockstor_config.png
```

Comment:
```suggestion
.. image:: /images/interface/system/services/replication_config.png
```

@FroggyFlox Sorry, just done another test build and it seems  I missed this one in my first review!
Otherwise we have the following:

![replication-with-rockstor-pic](https://github.com/rockstor/rockstor-doc/assets/2521585/53ad7c3d-21b5-4ea7-8fa9-c477b0a32c2f)

Reply:
Oh that is embarrassing for me... Sorry for missing that one but thank you so much for catching it.